Dockerfile.CI

```diff
@@ -60,3 +60,6 @@ RUN wget https://github.com/sigstore/cosign/releases/download/v2.2.4/cosign-linu
 
 RUN wget https://github.com/sigstore/rekor/releases/download/v1.3.6/rekor-cli-linux-amd64 -O /usr/bin/rekor-cli && \
     chmod u+x /usr/bin/rekor-cli
+
+ENV GOLANGCI_LINT_VERSION=1.59.0
+RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b /usr/bin v${GOLANGCI_LINT_VERSION}
```

Comment:
I would prefer downloading tarball from https://github.com/golangci/golangci-lint/releases/download/v1.59.1/golangci-lint-1.59.1-linux-amd64.tar.gz and extracting it. You never know what installation script is doing it's safer to keep it simple. We do the same for all other tools in this image.

Reply:
I tried extracting the tarball there was some issue with that executable file.

Reply:
/usr/bin/golangci-lint: cannot execute binary file: Exec format error

Reply:
Done